### PR TITLE
Feature: update styleguide

### LINKED ISSUE #22: Update repo to match updated coding style
- [HTML, JS, CSS/LESS] will now use 2 space indents
- [JS] will now use single quotes

### PR DESCRIPTION
**Pull request by jamesjwarren**
We've updated our JS/HTML/CSS styleguides, this updates this repo to match the new style. Fixes #22

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -5,84 +5,84 @@
...
 
-    <!-- build:template
-    <base href="<%= url %>">
-    /build -->
-    <!-- build:remove -->
-    <base href="/app/">
-    <!-- /build -->
+  <!-- build:template
+<base href="<%= url %>">
+/build -->
```
Comment (edwardoparearyee) [inline app/index.html]: here you've removed too many spaces

File: app/index.html [hunk 2]
```diff
@@ -5,84 +5,84 @@
...
-    <ng-view></ng-view>
+  <p class="text-right">App version: <strong app-version></strong></p>
+  <ng-view></ng-view>
 
-    <!--[if lt IE 8]>
-        <p>Your browser is <strong>ancient</strong> and will not work with this site. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-    <![endif]-->
+  <!--[if lt IE 8]>
+<p>Your browser is <strong>ancient</strong> and will not work with this site. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+<![endif]-->
```
Comment (edwardoparearyee) [inline app/index.html]: same here too many spaces removed

**Comment by edwardoparearyee** [comment]
other than the above all good :+1:

File: app/index.html (hunk 1 above)
Comment (jamesjwarren) [inline app/index.html]: thanks for spotting that, should be fixed now

**Comment by edwardoparearyee** [comment]
@jamesjwarren think you may have to disable the e2e tests for now, for some reason the node module is not running the tests on travis.

**Comment by edwardoparearyee** [comment]
@jamesjwarren can you fix this branch so we can merge it in, need it for new repos.

**Comment by jamesjwarren** [comment]
@edwardoparearyee should be good now

**Comment by edwardoparearyee** [comment]
:+1:  LGTM

**Pull request merged**